examples/stats/plot_regress_continuous.py

```diff
@@ -0,0 +1,62 @@
+"""
+========================================
+Regression on continuous data (rER[P/F])
+========================================
+
+This demonstrates how rERPs/regressing the continuous data is a
+generalisation of traditional averaging. If all preprocessing steps
+are the same and if no overlap between epochs exists and if all
+predictors are binary, regression is virtually identical to traditional
+averaging.
+If overlap exists and/or predictors are continuous, traditional averaging
+is inapplicable, but regression can estimate, including those of
+continuous predictors.
+
+References
+----------
+See Smith, N. J., & Kutas, M. (2015). Regression-based estimation of ERP
+waveforms: II. Non-linear effects, overlap correction, and practical
+considerations. Psychophysiology, 52(2), 169-189.
+
+Authors: Jona Sassenhagen <jona.sassenhagen@gmail.de>
+License: BSD (3-clause)
```

Comment:
Where's it go?

Also yes - it's a really simple method that's hard to explain because it's very similar to, but substantially superior to regular methods. Thus, very little code, much explanation.